addons/variable-manager/userscript.js

```diff
@@ -148,6 +149,19 @@ export default async function ({ addon, global, console, msg }) {
         if (existingVariableWithNewName) {
           label.value = this.scratchVariable.name;
         } else {
+          if (
+            !Blockly.Variables.nameValidator_.call(
+              null,
```

Comment:
Hmm, what's the null here?

Reply:
See https://github.com/LLK/scratch-blocks/blob/893c7e7ad5bfb416eaed75d9a1c93bdce84e36ab/core/variables.js#L297